### PR TITLE
[ready] beepsky smash change

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
balance: beepsky smash now summons imaginary beepskys that deal stamina damage instead of outright stunning
/:cl:

forgot to push the commit in the last pr

### CODE REVIEW

**Comment by wesoda25** [comment]
Give us video

**Comment by Angustmeta** [comment]
Lower metabolisation rate imo.

**Comment by Shadowflame909** [comment]
should make it still stun since you can run away from the holobeepsky

**Comment by ATH1909** [comment]
Good PR.

**Comment by Tlaltecuhtli** [comment]
@wesoda @shadow u can test it in local

File: code/datums/brain_damage/special.dm [hunk 1]
```diff
@@ -182,8 +182,71 @@
...
+
+/datum/brain_trauma/special/beepsky/on_life()
+	if(get_dist(owner, beepsky) >= 10 && prob(20))
+		QDEL_NULL(beepsky)
+		create_securitron()
+	if(owner.stat != CONSCIOUS)
+		if(prob(20))
+			playsound(owner, 'sound/voice/beepsky/iamthelaw.ogg', 50)
+		return
+	if(!beepsky || !beepsky.loc || beepsky.z != owner.z)
```
Comment (ShizCalev) [inline code/datums/brain_damage/special.dm]: if(QDELEDED(beekspy)

Comment (Tlaltecuhtli) [reply code/datums/brain_damage/special.dm]: wouldnt that cause problems if you change z level?

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -640,19 +640,42 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+/datum/reagent/consumable/ethanol/beepsky_smash/on_mob_life(mob/living/carbon/M)
+	M.Jitter(2)
+	if(M.has_trait(TRAIT_LAW_ENFORCEMENT_METABOLISM))
+		M.adjustStaminaLoss(-10, 0)
+		if(prob(20))
+			new /datum/hallucination/items_other(M)
+		if(prob(10))
+			new /datum/hallucination/stray_bullet(M)
+	..()
+	. = 1
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: lol just return true here

**Pull request merged**